src/OrchardCore.Modules/OrchardCore.Roles/Views/Admin/Edit.cshtml

```diff
@@ -15,14 +15,14 @@
 </form>
 
 <form asp-action="Edit" method="post" class="no-multisubmit">
-    <h5>@T["Edit Role Detail"]</h5>
```

Comment:
Why you change the header size?

Reply:
because we changed how the title appears. I thought it would look better if the header has a different size than the actual legend labels